### DISCUSSION TITLE
Unable to open AirFlow webserver UI

### DISCUSSION THREAD

**Discussion by MattSalminen** [apache/airflow#39699] [General]
### Apache Airflow version

2.9.1

### If "Other Airflow 2 version" selected, which one?

2.9.0+astro.1

### What happened?

When the docker container is running and I perform "astro dev start" or "astro dev restart", everything works properly. When I open the localhost:8080/home, it takes me to an error as shown here:

![airflow_error](https://github.com/apache/airflow/assets/139598105/fe9c9575-72c3-47c0-ae33-e9c39c3ae660)

### What you think should happen instead?

I open the UI successfully

### How to reproduce

Pretty much install astro, run the command to start airflow once docker is running and the airflow container is running.

### Operating System

Windows

### Versions of Apache Airflow Providers

astro

### Deployment

Official Apache Airflow Helm Chart

### Anything else?

using Astro CLI which may or may not need to be upgraded?

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.
